src/PathCompletionItem.ts

```diff
@@ -34,11 +34,11 @@ export class PathCompletionItem extends CompletionItem {
         const fragments = fileInfo.file.split('.');
         const extension = fragments[fragments.length - 1];
 
-        if (extension !== documentExtension) {
+        if (extension !== documentExtension && !isImport) {
```

Comment:
You are right :) Maybe we can make this if smarter? What do you think about making '.js'\'.jsx'\'.ts'\'.tsx' extensions an exception?